plugins/in_opentelemetry/opentelemetry_prot.c

```diff
@@ -1736,6 +1741,8 @@ int opentelemetry_prot_handle(struct flb_opentelemetry *ctx, struct http_conn *c
         }
     }
 
+    size_t tag_len = flb_sds_len(tag);
```

Comment:
Declaring middle of the function is not good coding style on Fluent Bit.
Could you declare a variable in the beginning of this function? And then, calculate length of tag here should be fine.

Reply:
Sure can. Should be pushed now.

Reply:
> Declaring middle of the function is not good coding style on Fluent Bit. Could you declare a variable in the beginning of this function? And then, calculate length of tag here should be fine.

Sure can. Should be pushed now.